assets/scss/_colors.scss

```diff
@@ -0,0 +1,7 @@
+@use 'uswds-core' as *;
+
+$grey-darker: color('base-darker');
+
+$blue-light: color('primary-light');
```

Comment:
I'd personally favor calling `color` directly rather than creating these variables, since:

1. Variables create a lot of redundancy
2. They're at risk of being incomplete (e.g. [`.footer` styles](https://github.com/GSA-TTS/identity-dev-docs/pull/478/files#diff-dd99657d7065f7185a4acbd6744dc46d4b9d13433c15b566c8e535e8184ff25dR5) call `color('primary-lightest');` directly, so why is that not a variable as well?)
3. Using [tokens](https://designsystem.digital.gov/design-tokens/color/overview/) is more semantic and avoids tying our code to specific colors, e.g. if we were to rebrand, "primary" is still applicable, but "blue" may not be